### ISSUE TITLE
Cannot expand long message due to special characters

### ISSUE THREAD

**Issue by ailjushkin** [reportportal/reportportal#287]
**Versions:**

- API Service: 3.3.1; 
- Service Index: 3.1.1; 
- Jira Service: 3.3.0; 
- Authorization Service: 3.3.0; 
- Service UI: 3.3.0;

**Problem:**

Due to some complex logging, UI log elements became absolutely not functional.

![rp-ui-error](https://user-images.githubusercontent.com/3462506/33600465-5b77c8b4-d9ba-11e7-9844-c35bf828d12e.png)

**Example of log messages:**

```
> GET http://xxxxxxxxxx:123/;oaisjdpio;jasd/v1/;lakjsd;lkjasd/7076/lkjhasdfkjhg/asdasdasd?limit=0
> User-Agent: python-requests/2.12.4
> Accept-Encoding: gzip, deflate
> Accept: */*
> Connection: keep-alive
> Content-Type: application/json
> Charset: UTF-8
> asdasdasd: 1
> sadflkjhsdklfjorization: ahdfkljhalskjdhfklas
> ps-timezone: Europe/Moscow
> Cookie: asdasdtoken=klsdjfal;kjsd;lfkjasdf
>
* Elapsed time: 0:00:00.772647
>
< 500 Internal Server Error
< Server: nginx/1.11.6
< Date: Tue, 05 Dec 2017 09:22:14 GMT
< Content-Type: application/json; charset=UTF-8
< Content-Length: 9978
< Connection: keep-alive
< asdasdsso-pid: <0.29905.2>
< pstxid: f2c5b15e442a1467578e5cff60651366
< x-forwarded-entry: asdsda
< asdasdsso-token: lkasdjf;lkajs;lfd
< set-cookie: asdasdtoken=kjhagdkfjhgakfjhgsdfkjhasdjf; Version=1; Domain=net.asdasdsa.ru; Path=/; HttpOnly
< asdasdsso-notification: password_expiration=,password_type=PERMANENT,password_attempts=1,password_max_attempts=10
< charset: UTF-8
< ps-auth-appl-code: фывфы
< ps-auth-appl-type: ффывфыв
< ps-auth-user: asdasdas
< ps-timezone: Europe/Moscow
< pstraceenable: 1
< subscriberId: 7076
< x-forwarded-for: 1111.52.8, 1111.168.1.101
< x-real-ip: 1111.52.8
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_;oaisjdpio;jasd.v1.;lakjsd;lkjasd-018a6a29-e075-45ec-b0d4-221be4443e42_asdasdasdAvailableForActivate_direct_in: 2017-12-05T12:22:14.086+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_asdasdasd.asdadas;lakjsd;lkjasd-asdasdasdAvailableForActivate_in: 2017-12-05T12:22:14.086+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_asdasdasd.asdadas;lakjsd;lkjasd-asdasdasdAvailableForActivate_routing_gfRouter--begin: 2017-12-05T12:22:14.087+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_asdasdasd.asdadas;lakjsd;lkjasd-asdasdasdAvailableForActivate_routing_gfRouter--end: 2017-12-05T12:22:14.087+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_asdasdasd.asdadas;lakjsd;lkjasd-asdasdasdAvailableForActivate_routing_gfRouter--routing_key_not_found: 2017-12-05T12:22:14.087+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_asdasdasd.asdadas;lakjsd;lkjasd-JAVA-com.peterservice.;oaisjdpio;jasd.asdasdasd.asdasdaselfcare.SelfcareAccessLimiter_in: 2017-12-05T12:22:14.087+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_asdasdasd.asdadas;lakjsd;lkjasd-JAVA-com.peterservice.;oaisjdpio;jasd.timezones.TimeZonePreprocessor_in: 2017-12-05T12:22:14.087+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_asdasdasd.asdadas;lakjsd;lkjasd-JAVA-com.peterservice.;oaisjdpio;jasd.asdasdasd.asdasdalocation.SwitcherPreprocessorForGetter_in: 2017-12-05T12:22:14.088+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_asdasdasd.asdadas;lakjsd;lkjasd-JAVA-com.peterservice.;oaisjdpio;jasd.ps.bis.;lakjsd;lkjasd.SubscriberIdentificationPreprocessor_in: 2017-12-05T12:22:14.088+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_asdsadflkjhsdklfjenticator_authenticateByToken_in: 2017-12-05T12:22:14.091+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_asdsadflkjhsdklfjenticator_authenticateByToken_out: 2017-12-05T12:22:14.091+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_;oaisjdpio;jasd.v1.ccmpim-lkjhasdfkjhgAvailableForActivateGetPIM_in: 2017-12-05T12:22:14.091+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_;oaisjdpio;jasd.v1.ccmpim-lkjhasdfkjhgAvailableForActivateGetPIM_out: 2017-12-05T12:22:14.237+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_;oaisjdpio;jasd.v1.;lakjsd;lkjasd-asdasdasdAvailableForActivate_out: 2017-12-05T12:22:14.238+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_asdasdasd.asdadas;lakjsd;lkjasd-asdasdasdAvailableForActivate_out: 2017-12-05T12:22:14.238+03:00
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_;oaisjdpio;jasd.v1.;lakjsd;lkjasd-asdasdasdAvailableForActivate_DEBUG: {"frontendHost":"1111.168.1.203:22003"}
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_asdasdasd.asdadas;lakjsd;lkjasd-asdasdasdAvailableForActivate_DEBUG: {"frontendHost":"1111.168.1.203:22046"}
< askldjhlkjhsadf_f2c5b15e442a1467578e5cff60651366_;oaisjdpio;jasd.v1.ccmpim-lkjhasdfkjhgAvailableForActivateGetPIM_DEBUG: 
< askldjhlkjhsadfId: f2c5b15e442a1467578e5cff60651366
< askldjhlkjhsadfLog: timestamp: 1512465734.241; uri: GET /;oaisjdpio;jasd/v1/;lakjsd;lkjasd/7076/lkjhasdfkjhg/
<
```

**Comment by evjlobanova** [comment]
@ailjushkin we fixed it in 3.3.2, please update your version

**Comment by ailjushkin** [comment]
@evjlobanova great, that works, thank you all
